app/views/govuk_component/docs/button.yml

```diff
@@ -48,6 +48,10 @@ examples:
       href: "#"
       start: true
       info_text: "Sometimes you want to explain where a user is going to."
+  with_margin_bottom:
+    data:
```

Comment:
Maybe worth adding a description here, similar to the description of this PR? Just to explain why this option is here.